app/src/main/java/com/techcourse/dao/UserDao.java

```diff
@@ -14,7 +11,16 @@
 public class UserDao {
 
     private static final Logger log = LoggerFactory.getLogger(UserDao.class);
-    private static final UserRowMapper USER_ROW_MAPPER = new UserRowMapper();
+    private static final RowMapper<User> USER_ROW_MAPPER;
+
+    static {
+        USER_ROW_MAPPER = resultSet -> new User(
+                resultSet.getLong("id"),
+                resultSet.getString("account"),
+                resultSet.getString("password"),
+                resultSet.getString("email")
+        );
+    }
```

Comment:
변수로 바로 할당하지 않고 `static` 블럭에서 진행한 이유가 있나요?

```suggestion
    private static final RowMapper<User> USER_ROW_MAPPER = resultSet -> new User(
                resultSet.getLong("id"),
                resultSet.getString("account"),
                resultSet.getString("password"),
                resultSet.getString("email")
        );
    }
```